### ISSUE TITLE
[Documentation] Relationship between "pleaserun" and "fpm"

### ISSUE THREAD

**Issue by shevegen** [jordansissel/pleaserun#130]
Hello Jordan (and everyone else),

Today (just now, really) I noiced pleaserun.

Would it be possible to add a subsection about the relationship between fpm and pleaserun, if possible? Right now I am trying to figure this out.

Thanks anyway!

**Comment by jordansissel** [comment]
Certainly! I am actually working on new release of fpm that will include some pleaserun+fpm docs today.

Until there's a longer-form document, here is what I can tell you:

* fpm: I want FPM to be the easiest way to satisfy this statement, "I want to create an installable package"

* pleaserun: I want pleaserun to be the easiest way to satsify the statement, "I want to create a service I can start and stop"

Since pleaserun is newer, I'll give you an example.

With `fpm` sometimes I have given an example using redis. To elaborate, you can make a redis package like this:

* download redis source code
* build redis from source (./configure, make, whatever)
* use fpm to package: `fpm -s dir -t rpm -n redis ...`

The next step with something like redis is to actually make it a service you can start, stop, restart, and enable on system boot. To achieve this, you need to know what service manager your OS is using. On newer Fedora, CentOS 7, etc, you need to know to use systemd; on Ubuntu (16.04, etc), CentOS 6, and Amazon Linux, you need to know to use upstart.  On older systems, you may need to use sysv. On OSX you use launchd.

I was overwhelmed and frustrated with packaging, so I started working on fpm years ago. Similarly, I was overwhelmed and frustrated by service managers (systemd, upstart, etc), so I wrote pleaserun.

----

Now, you can actually use pleaserun+fpm together! And it's a good timing you asked about this because I wrote some pleaserun+fpm docs a few minutes ago -- http://fpm.readthedocs.io/en/latest/source/pleaserun.html
